src/main/docker/Dockerfile

```
@@ -1,20 +1,13 @@
-FROM openjdk:17-oracle as builder
-
-WORKDIR /app
-COPY target/plugin-health-scoring.jar /app/
-
-RUN java -Djarmode=layertools -jar plugin-health-scoring.jar extract
-
-FROM azul/zulu-openjdk:17.0.2-jre-headless
+FROM eclipse-temurin:17.0.4.1_1-jre
 
 RUN groupadd -r phs && useradd -r -g phs phs
 USER phs:phs
 
 WORKDIR /app
 
-COPY --from=builder /app/dependencies/ ./
-COPY --from=builder /app/spring-boot-loader/ ./
-COPY --from=builder /app/snapshot-dependencies/ ./
-COPY --from=builder /app/application/ ./
+COPY target/application/ ./
+COPY target/dependencies/ ./
+COPY target/snapshot-dependencies/ ./
+COPY target/spring-boot-loader/ ./
 
 ENTRYPOINT ["java","org.springframework.boot.loader.JarLauncher"]
```

```suggestion
COPY target/plugin-health-scoring.jar /app/plugin-health-scoring.jar

ENTRYPOINT ["java", "-jar", "/app/plugin-health-scoring.jar"]
```

the layerisation of jar is nice, but it is an optimization that we do not really need. Since it's failing, let's keep a "classic" Dockerfile model and we'll improve later (tip: multi-stage should use the same JRE, and we use temurin instead of azul + openjdk is deprecated in favor of temurin).

---

I don't understand why the application and dependencies folders are copied without issue while the snapshot-dependencies one is failing. Is it because of the folder name?